sdk/containers/azcontainerregistry/cloud_config.go

```diff
@@ -24,3 +24,7 @@ func init() {
 		Audience: "https://management.core.windows.net/",
 	}
 }
+
+var defaultCloud = cloud.Configuration{
+	Services: map[cloud.ServiceName]cloud.ServiceConfiguration{ServiceName: {Audience: defaultAudience}},
```

Comment:
All the cloud configurations defined in this file work. This default cloud is just created to set default audience recommended by service team if user does not set any audience (just because Go use cloud configuration for audience config).

Reply:
I get it now, thanks for the explanation.